server/api/scenarios.post.ts

```diff
@@ -1,12 +1,12 @@
 import type { NewScenarioResponse } from "@/types/apiResponseTypes";
-import { runScenario } from "@/server/handlers/scenarios";
+import { newScenario } from "@/server/handlers/scenarios";
 import { defineRApiEventHandler } from "@/server/utils/defineRApiEventHandler";
 
 export default defineRApiEventHandler(
   async (event): Promise<NewScenarioResponse> => {
     const body = await readBody(event);
     const parameters = body.parameters;
-    const newScenarioResponse = await runScenario(parameters, event);
+    const newScenarioResponse = await newScenario(parameters, event);
```

Comment:
We're keeping the runScenario function, but now we're wrapping it in a new function called newScenario which does more than just running the scenario.